src/index.js

```diff
@@ -8,272 +8,358 @@ const _ = require("lodash");
 const {
   getAllTransformations,
   getAllLibraries,
-  createTransformer,
+  createTransformation,
   createLibrary,
-  updateTransformer,
+  updateTransformation,
   updateLibrary,
   testTransformationAndLibrary,
   publish,
 } = require("./apiCalls");
 
 const testOutputDir = "./test-outputs";
-const uploadTestArtifact = core.getInput("uploadTestArtifact") || false;
+const uploadTestArtifact =
+  Boolean(core.getInput("uploadTestArtifact")) || false;
```

Comment:
This is not correct right, the statement. Boolean will treat any truthy value to be `true` meaning `Boolean("false")` would result in `true`. Can you check and fix it ?

Reply:
All the values pulled through core.getInput("abc") would be string and result in `true`

Reply:
Why not simply do this:
```suggestion
const uploadTestArtifact = core.getInput("uploadTestArtifact").toLowerCase() == "true"
```